backend/frontend/tests/test_card_cancel_review_request_button.py

```diff
@@ -32,7 +32,7 @@ def make_project_card(self, project_submission_type):
     def test_cancel_review_request_button_moves_project_card_to_ip_column(self):
         self.make_project_card(ContentItem.LINK)
 
-        self.page.click("text=Cancel review request")
+        self.page.click("text=Cancel review request", timeout=0)
```

Comment:
This is a bit sketchy. Timeouts exist for a reason. What if one of our things just seriously doesn't work, and we get something that just loads forever? 

Timeouts save lives. 

This is a hack that would make the tests pass. But Django has a way of dealing with static resources. 

Can we not just do things the Django way?